ecl/eclcmd/eclcmd_common.hpp

```diff
@@ -159,6 +159,9 @@ typedef IEclCommand *(*EclCommandFactory)(const char *cmdname);
 #define ECLOPT_PROCESS_S "-p"
 #define ECLOPT_SOURCE_PROCESS "--source-process"
 
+#define ECLOPT_PATH "--path"
+#define ECLOPT_INC_THOR_SLAVE_LOGS "--incThorSlaveLogs"
+#define ECLOPT_PROBLEM_DESC "--problemDescription"
```

Comment:
Perhaps just "--description"?  Unless there are possibly other types of description.

Reply:
Changed